<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowAc</class>
 <widget class="QWidget" name="ShowAc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="wind">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="work_state_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="cur_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;华文彩云&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="home_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;华文彩云&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="money_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>消耗金额</string>
   </property>
  </widget>
  <widget class="QLabel" name="home_id_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QLabel" name="wind_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前风速</string>
   </property>
  </widget>
  <widget class="QLabel" name="cur_temp_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="temp_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="temp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;华文彩云&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="work_state">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;华文彩云&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/ac_window.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <zorder>energy_label</zorder>
   <zorder>energy</zorder>
  </widget>
  <widget class="QLabel" name="energy">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;华文彩云&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="energy_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>能耗</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>wind</zorder>
  <zorder>work_state_label</zorder>
  <zorder>cur_temp</zorder>
  <zorder>home_id</zorder>
  <zorder>money_label</zorder>
  <zorder>home_id_label</zorder>
  <zorder>wind_label</zorder>
  <zorder>cur_temp_label</zorder>
  <zorder>temp_label</zorder>
  <zorder>temp</zorder>
  <zorder>work_state</zorder>
  <zorder>money</zorder>
  <zorder>energy</zorder>
  <zorder>energy_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
